cluster/pulumi/sv-runbook/src/installNode.ts

```diff
@@ -327,6 +328,7 @@ async function installSvAndValidator(
     maxVettingDelay: networkWideConfig?.maxVettingDelay,
     logLevel: svConfig.logging?.appsLogLevel,
     additionalEnvVars: svAppAdditionalEnvVars,
+    additionalJvmOptions: getAdditionalJvmOptions(svConfig.svApp?.additionalJvmOptions),
```

Comment:
we genuinely did not enable jmx for the sv runbook up until now. That seems like a mistake to me.